portfolio/src/main/webapp/background.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title> Carol's Portfolio </title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"> </script>
+  </head>
+  <body>
+    <div class="container">
+      <div class="menu">
+        <ul>
+            <li class="logo"> <a href="index.html"> <img src="images/logo.jpg"> </a> </li>
+            <li> <a href="index.html"> About Me </a> </li>
+            <li class="currentLink"> <a href="background.html" style="color:white"> Background </a> </li>
+            <li> <a href="photos.html"> Photos </a> </li>
+            <li> <a href="contact.html"> Contact Me </a> </li>
+        </ul>
+      </div>
+    </div>
+    <h1> Background </h1>
+    <p> I am a computer science major at Caltech, expecting to graduate in June 2022. <br> 
+      So far, some of the theoretical CS classes I have taken are algorithms, decidability and tractability, and learning systems. <br> 
+      Some of the practical CS classes I've taken are data structures, software design, and machine learning. <br> 
+      I'm excited to be taking some more in-depth CS classes in my next two years at Caltech, including operating systems, information theory, and databases. 
+    </p>
+    <p>
+    I've been lucky to have had a few internships before and during college, <br> 
+      including a summer research project at the National Institutes of Health studying Multiple Sclerosis, <br> 
```

Comment:
The `I've` should be two spaces to the right to be align with the line from below.